app/components/searchBar/index.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { screen, render, fireEvent, getByTestId, waitFor } from '@testing-library/react';
+import SearchBar from '.';
+import userEvent from '@testing-library/user-event';
+
+describe('Input Value', () => {
+  it('renders a input field', () => {
+    render(<SearchBar placeholder="Search" defaultValue="" onChangeValue={() => {}} />);
+    const searchBar = screen.getByTestId('searchBar');
+    expect(searchBar).toBeInTheDocument();
+  });
+});
+
+describe('Not Empty', () => {
+  it('checks input value to be empty', () => {
+    render(<SearchBar placeholder="" defaultValue="" onChangeValue={() => {}} />);
+    const inputElement = screen.getByTestId('searchBar');
+    expect(inputElement).toBeInTheDocument();
+  });
+});
+
+describe('Accepting Letters', () => {
+  it('checks input letter', () => {
+    render(<SearchBar placeholder="" onChangeValue={jest.fn((value) => {})} />);
+    const letters = 'abcdefghizklmnopqrstuvwxyz';
+    const input = screen.getByTestId('searchBar') as HTMLInputElement;
+    fireEvent.change(input, { target: { value: letters } });
+    expect(input).toHaveValue(letters);
+  });
+});
+
+describe('Accepting Symbols', () => {
+  it('check input symbols', () => {
+    render(<SearchBar placeholder="" defaultValue="" onChangeValue={() => {}} />);
+    const symbols = '!@#$%^&*()_+-=';
+    const inputSymbols = screen.getByTestId('searchBar') as HTMLInputElement;
+    fireEvent.change(inputSymbols, { target: { value: symbols } });
+    expect(inputSymbols).toHaveValue(symbols);
+  });
+});
```

Comment:
Have one `describe` block for checking allowed values and separate them using `it` blocks